topology/probes/neutron.go

```diff
@@ -360,6 +360,24 @@ func (mapper *NeutronProbe) OnNodeUpdated(n *graph.Node) {
 
 // OnNodeAdded event
 func (mapper *NeutronProbe) OnNodeAdded(n *graph.Node) {
+	name, _ := n.GetFieldString("Name")
+	attachedMAC, _ := n.GetFieldString("ExtID.attached-mac")
+	if attachedMAC == "" && strings.HasPrefix(name, "tap") {
+		qvo := strings.Replace(name, "tap", "qvo", 1)
+		qvoNode := mapper.graph.LookupFirstNode(graph.Metadata{"Name": qvo, "Type": "veth"})
+		if qvoNode != nil {
+			tr := mapper.graph.StartMetadataTransaction(n)
+			if attachedMAC, _ = qvoNode.GetFieldString("ExtID.attached-mac"); attachedMAC != "" {
+				tr.AddMetadata("ExtID.attached-mac", attachedMAC)
```

Comment:
Could you add Neutron sub metadata too ?

Reply:
mapper.enhanceNode(n) will  add Neutron sub metadata automatically below
It will retrieve neutron port  info through "ExtID.attached-mac" in "retrieveAttributes" function
so we add "ExtID.attached-mac" is eough, now new tap node can get all metadata with ExtID and Neutron

Reply:
ok thanks for the explanation, going to test it and merge it if OK

Reply:
OK for me, waiting the CI result